src/PasswordHasher.php

```diff
@@ -17,10 +18,10 @@ class PasswordHasher
 
     /**
      * @see https://www.php.net/manual/en/function.password-hash.php
-     * @param int|string $algorithm
+     * @param int $algorithm
      * @param array|null $parameters
      */
-    public function __construct($algorithm = PASSWORD_DEFAULT, array $parameters = null)
+    public function __construct(int $algorithm = PASSWORD_DEFAULT, array $parameters = null)
```

Comment:
No, that should be left as is. See https://github.com/yiisoft/security/issues/5. That probably should be added to phpdoc.